readme.md

```diff
@@ -86,7 +86,7 @@ There are two ways to run Renovate on Azure DevOps:
 Go to the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organization.
```

Comment:
By the way, it looks like we're still using the _old_ syntax in the code:

Example from `lib/workers/repository/dependency-dashboard.spec.ts`:

```ts
      describe('PackageFiles.getDashboardMarkdown()', () => {
        const note =
          '> **Note**\n> Detected dependencies section has been truncated\n';
        const title = `## Detected dependencies\n\n`;
```

And from `lib/workers/repository/package-files.ts`

```ts
  static getDashboardMarkdown(maxLength: number, setHeader = true): string {
    const note =
      '> **Note**\n> Detected dependencies section has been truncated\n';
    const title = `## Detected dependencies\n\n`;
```

Reply:
If it's markdown on GitHub for our docs (e.g. this PR contents) then we can use their custom syntax, but if it's markdown in the bot's PRs or issues then we should use generic markdown by default. Is `[!NOTE]` generic markdown or github-specific?

Reply:
### Looks like the Markdown goes in the Dependency Dashboard issue

I think the code is for the "Dependency Dashboard is truncated due to hitting a platform length limit" message. I don't know if this part of the code is only reached by the GitHub-platform though.

### Generic Markdown or GitHub-specific?

The [CommonMark spec 0.30 for Block Quotes](https://spec.commonmark.org/0.30/#block-quotes) does not have `alert` or `note` alerts/admonitions. The GitHub Docs say, (I **bolded** the key part):

> Alerts are an **extension of the blockquote syntax** that you can use to emphasize critical information. On GitHub, they are displayed with distinctive colors and icons to indicate the importance of the content.

GitHub says they're extending something, so that means it's not in the default set of allowed Markdown syntaxes.

Reply:
In that case the syntax you highlighted is not "old" - it's just generic/standard, so I think we can resolve this?

Reply:
## Decipher the code's adventure

> In that case the syntax you highlighted is not "old" - it's just generic/standard, so I think we can resolve this?

It all depends on what the final output is of this code:

```ts
        const note =
          '> **Note**\n> Detected dependencies section has been truncated\n';
```

### Option 1, code expands with `>` in output

I see `>` characters in the code, so I suspect this is the output:

> **Note**
> Detected dependencies section has been truncated

If that what happens, we should use the _new_ `> [!NOTE]` syntax.

### Option 2, code expands without `>` in output

But if I'm wrong, and the code expands to:

**Note**
Detected dependencies section has been truncated

We don't need to do anything.

Reply:
True. We shouldn't be spilling GitHub-specific syntax into other platforms. But I'm not sure if that's the case or not

Reply:
I don't know at all... I'll let a proper programmer figure it out. 🙃

Reply:
Let's treat it as a separate topic nto this _docs_ update